.gitignore

```diff
@@ -33,5 +33,6 @@
 .settings/
 
 wordlist.dic
+config.txt
```

Comment:
What's this `config.txt` about? :-)

Reply:
An automation file for RLTest. @MeirShpilraien can explain more in detail.

Reply:
Got you. The name is too generic. I think RLTest should have a different configuration name.

Reply:
@iddm We can deprecate this name and come up with another one more specific to RLTest. maybe `rltest_config.txt`? Personally I think maybe RLTest should also look for this configuration file under some user directory (maybe something like `~/.rltest/rltest_config.txt`) and not just under the current directory.

Reply:
Yes, indeed, much better than just `config.txt` :-) But even better, just as you said, a config directory, I think. However, we should use something like `XDG_CONFIG_DIR` or something if it is available; if not, then just use `$HOME/.config/rltest` for it.